test/TestProjects/SubscriptionExtensions/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -37,15 +37,16 @@ private static OvensRestOperations GetOvensRestOperations(ClientDiagnostics clie
 
         /// <summary> Lists the Ovens for this Azure.ResourceManager.Core.SubscriptionOperations. </summary>
         /// <param name="subscription"> The <see cref="SubscriptionOperations" /> instance the method will execute against. </param>
+        /// <param name="statusOnly"> statusOnly=true enables fetching run time status of all Virtual Machines in the subscription. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <return> A collection of resource operations that may take multiple service requests to iterate over. </return>
-        public static AsyncPageable<Oven> ListOvenAsync(this SubscriptionOperations subscription, CancellationToken cancellationToken = default)
+        public static AsyncPageable<Oven> ListOvenAsync(this SubscriptionOperations subscription, string statusOnly = null, CancellationToken cancellationToken = default)
```

Comment:
The `ListAllAsync` function at Line 61 has an optional parameter `statusOnly` which defined in its corresponding swagger.
This PR includes a change that also add this optional parameter to the signature of this public wrapper function.

This change is breaking some of the test cases in `TestProjectTests.cs` because it is asserting we have only `2` parameters.

Reply:
I believe those tests are fixed and we can resolve this conversation?